lib/resources/r4/financial_transaction.yaml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Let's document the binding on Money.currency just like what we did in Account

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/7032a15b7d3e5ed683f2b305d95e072796957d99#diff-5125c86cde87f8439cb9d1bb859194e9R30

Reply:
+1